include/faabric/scheduler/MpiWorld.h

```diff
@@ -195,6 +195,10 @@ class MpiWorld
 
     std::shared_ptr<InMemoryMpiQueue> getLocalQueue(int sendRank, int recvRank);
 
+    void setLocalQueues();
+
+    int getKeyForRanks(int sendRank, int recvRank);
```

Comment:
Using the word "key" is a little confusing as it's not a map any more. Perhaps `getArrayIndexForRanks` or something.

This should also be `private` as it's an internal method. At the moment everything on this class is `public` which should be fixed as well (but in another PR).